src/main/kotlin/com/mdrobnak/lalrpop/injectors/LpRustActionCodeInjector.kt

```diff
@@ -32,16 +30,37 @@ class LpRustActionCodeInjector : MultiHostInjector {
             .mapNotNull { it.getSelectedType() }
         val returnType = nonterminal.typeRef?.text ?: nonterminal.nonterminalName
 
+        val grammarDecl = PsiTreeUtil.findChildOfType(context.containingFile, LpGrammarDecl::class.java)
+
+        val grammarParams = grammarDecl?.grammarParams
+        val grammarParametersString = if (grammarParams != null && grammarParams.grammarParamList.isNotEmpty()) {
+            grammarParams.grammarParamList.joinToString(separator = "") { "${it.getName()}: ${it.typeRef.text}," }
+        } else {
+            ""
+        }
```

Comment:
Can this be simplified to work in the same way as `grammarTypeParamsString` and `grammarWhereClausesString`?